quickjs.c

```diff
@@ -58038,6 +58098,13 @@ uintptr_t js_std_cmd(int cmd, ...) {
         *pv = ctx->error_back_trace;
         ctx->error_back_trace = JS_UNDEFINED;
         break;
+    case 3: // GetStringKind
```

Comment:
Any reason to expose it like this vs an actual API? I assume because it should be transparent to users?